### PR TITLE
fix moderated sessions setter and getter

### PR DESCRIPTION
**Pull request by NajiObeid**
Part of https://github.com/gravitational/cloud/issues/1701

This fixes the license's moderated sessions setter and getter

### CODE REVIEW

**Review by rosstimothy — COMMENTED** [review]
Can we add test cases to `license_test` that only have one feature enabled per test case such that we can catch future copy/paste issues like this?

**Review by zmb3 — APPROVED** [review]

**Review by rosstimothy — APPROVED** [review]

**Comment by nklaassen** [comment]
We haven't given out any licenses with moderated sessions support yet have we? If anyone has one, it probably won't actually be enabled in the license and they'll need a new one

**Comment by NajiObeid** [comment]
@nklaassen unless they got it with desktop access 🥲

File: api/types/license_test.go [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+		name        string
+		setter      func(License, Bool)
+		getter      func(License) Bool
+		unsetValues [](func(License) Bool)
+	}{
+		{
+			name:   "Set ReportsUsage",
+			setter: License.SetReportsUsage,
+			getter: License.GetReportsUsage,
+			unsetValues: []func(License) Bool{
```
Comment (codingllama) [inline api/types/license_test.go:36]: Could we have a single list with all license getters and have the test skip the current getter from it? It's easier to review a single list than having to eyeball the diffs in every test case.

**Review by codingllama — COMMENTED** [review]
Well spotted! Thanks for the fix.

**Review by codingllama — APPROVED** [review]

**Comment by zmb3** [comment]
Do we need a v9 backport for this?

**Pull request merged**